docs/user-documentation/objects.md

```diff
@@ -93,3 +93,21 @@ All nodes can contain any number of media.  The media for any node can be manage
 the "Members" tab, Actions can be performed in bulk using the check boxes and Actions drop down.
 
 ![Media tab](../assets/islandora_8_derivatives_sample.png)
+
+## Display Hints
+
+Media are displayed in Islandora 8 via "view modes". View modes are Drupal's name for alternative ways to present content to users. In Islandora, the view mode determines whether a media is displayed using the OpenSeadragon viewer or the PDFjs viewer, for example. View modes can be set explicitly in the node edit form, or you can configure Islandora to use a specific view mode for all media with a specific Mime type.
+
+At the node level, in the "Display hints" field, simply check the display mode you want to use for that node:
+
+![Display hints](../assets/display_hints.png)
+
+The selected view mode will then be used when the node is viewed.
+
+At a global level, you can create a [Context](context.md) that will detect the media's Mime type and use the configured view mode automatically. To do this, go to Drupal's "Structure" menu, then "Context" and create a new Context. Then, add a "Media has Mime type" condition and specify the Mime type, and add a "Change View mode" Reaction that selects the desired view mode:
```

Comment:
I think the most appropriate way to do this is to modify the existing Context condition that already exists for the display view mode. E.g. if we want use the PDF mimetype to trigger the PDFjs view mode they should edit the existing one (/admin/structure/context/pdfjs) and add the mimetype condition. As long as "Require all conditions" is **not** selected, either the mimetype or display mode radio button will trigger the display mode update.

Reply:
This makes it more clear to an admin that _we already have a condition to do this_ but we are adding an additional capability that alleviates us from needing to select the view mode explicitly.

Perhaps the more common example would be openseadragon where we want to use the mimetype for JP2 but we can manually select it for regular JPEGs if desired.

Reply:
I agree that there are benefits to modifying the existing context, but I think whether or not an admin decides on doing that or creating a new context is probably a matter of personal preference. Your suggested example of an override is good.

Reply:
I'm happy to document modifying the existing context but I'd like to mention that there are two ways of doing it. But, that might be too far into the weeds.

Reply:
That is fine. I just didn't want new admins to create a new context without realizing that a similar  one already exists, especially because creating a separate one could lead to the context ordering issue mentioned in the other thread. 

Perhaps you could lead with the modification example and then follow-up with the "you can add an additional, separate, condition; but keep in mind that the order of context processing may cause one to override the other" (or something to that effect).

Reply:
Both approaches are described now.